src/index.js

```diff
@@ -8,7 +8,6 @@ const fs = require("fs"),
     cors = require("cors"),
     passport = require("passport"),
     errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
```

Comment:
Good job on the removal of mongoose setup

Reply:
thanks